benefits/static/css/styles.css

```diff
@@ -163,6 +167,16 @@ footer {
   margin-bottom: 2rem;
 }
 
+.container.content h1 {
+  font-size: 24px;
+  line-height: 30px;
```

Comment:
I think these rules belong in the [`h1` block](https://github.com/cal-itp/benefits/blob/dev/benefits/static/css/styles.css#L37) up at the top of the document

Reply:
Add `letter-spacing: 0.05em;`